challenges/objects-arrays.js

```diff
@@ -40,26 +61,38 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":7,"first_name":"Robbi","university":"Salem University","email":"rbrister6@redcross.org"},
 {"id":8,"first_name":"Colline","university":"Coastal Carolina University","email":"cbrosh7@alibaba.com"},
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
-{"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
+{"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}];
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
-
+for(let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+  universities.sort();
+console.log(universities);
+}
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name);
+  contactInfo.push(graduates[i].email);
+  }
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for(let i = 0; i < graduates.length; i++) {
+    if (graduates[i].university === "Uni") {
```

Comment:
This will return `true` for a university called "Uni" but not for a university with a name that _contains_ "Uni." Use `.includes()` instead (it works on Strings too, not just Arrays!)